src/ProductConstructionService/ProductConstructionService.Api/TestController.cs

```diff
@@ -1,16 +1,32 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Azure.Storage.Queues;
 using Maestro.Data;
 using Microsoft.AspNetCore.Mvc;
+using ProductConstructionService.Api.Queue;
+using ProductConstructionService.Api.Queue.WorkItems;
 
 namespace ProductConstructionService.Api;
 [Route("test")]
-public class TestController(BuildAssetRegistryContext dbContext) : Controller
+public class TestController(BuildAssetRegistryContext dbContext, QueueMessageSenderFactory queueInjectorFactory, QueueServiceClient client) : Controller
 {
-    [HttpGet]
-    public IActionResult Index()
+    private readonly QueueMessageSenderFactory _queueInjectorFactory = queueInjectorFactory;
+    private readonly BuildAssetRegistryContext _dbContext = dbContext;
+
+    [HttpGet("1")]
+    public async Task<IActionResult> Index()
+    {
+        var queueInjector = _queueInjectorFactory.Create<TextWorkitem>();
```

Comment:
Injector sounds off? Maybe a producer or sender something?

Reply:
if we call the queue `pcs-jobs`, them maybe we can call it `PcsJobsProducer` and `PcsJobsProducerFactory`

Reply:
Yeah, let's do that

Reply:
But we should also maybe unify WorkItem and Job terminology.

It can be either so even the queue can be pcs-workitems or so

Reply:
Great, let's do that